extensions/README.md

```diff
@@ -0,0 +1,152 @@
+# Blade Extensions
+
+There are several ways that you can extend `blade` CLI.  Most simply you can add custom `commands` and `templates`.
+But likely if you want to customize a developer's experience with `blade` to do enable some new developer workflow, you
```

Comment:
What about "to enable a fully-customized liferay workspace workflow, tailored for your requirements"

Reply:
+1